### ISSUE TITLE
Input on 4.6 Charts, diagrams, math, and formulas

### ISSUE THREAD

**Issue by jonaslil** [w3c/publ-a11y#485]
## Logic issue

In the EPUB techniques, `accessibilityFeature` `"describedMath"` is used in 4.6.3 step 5 to identify the presence of math content, but it is not used in step 4 as an indication of accessible math. Shouldn't it be included there as well?

## Specificity of explanations

In my view, LaTex and MathML for math content are so different that they should be expressed as separate explanations. For example, a screen reader user who doesn’t know LaTeX notation won’t find it helpful at all. However, I understand the intention is to keep things non-technical and to limit the number of explanations, so I won’t insist on this point.

## Discrepancy between Display guide and Techniques

The explanation "Visualized data also available as non-graphical data" in the ONIX techniques is missing in the Display guidelines.

## Phrasing and structure of explanations
Current: Charts and diagrams are present and described by extended descriptions
Suggested: Charts and diagrams are present and have extended descriptions
(Avoids the repetition in "described by ... description.")

Current: Charts and diagrams have extended descriptions Accessible math content Accessible chemistry content
Suggested: Split these into separate list items.
- Charts and diagrams have extended descriptions
- Accessible math content
- Accessible chemistry content

Current: Accessibility of formulas, charts, math, and diagrams not identified as being accessible
Suggested revision:
- Either: Formulas, charts, math, and diagrams not identified as being accessible
- Or: Accessibility of formulas, charts, math, and diagrams not identified

**Comment by GeorgeKerscher** [comment]
The Rich content heading replaces the old one. It has been revised.Will close the issue. If not resolved, please file a new issue.
